phones/tests/models_tests.py

```diff
@@ -122,13 +123,11 @@ def test_create_second_realphone_for_user_raises_exception(
     RealPhone.objects.create(user=phone_user, verified=True, number="+12223334444")
     mock_twilio_client = mocked_twilio_client
     mock_twilio_client.messages.create.assert_called_once()
-    try:
+    mock_twilio_client.reset_mock()
```

Comment:
I don't know of any case where you would want to create a copy of a mock, I don't think they work that way!

The reset is so that `mock_twilio_client.messages.assert_not_called()` will be true later. Without it, it will still have been called once from creating the original number.

Reply:
This was clearer in conversation. The issue is this line:

```
mock_twilio_client = mocked_twilio_client
```
This looks like a soft copy, but mocks don't work that way, it is more like making an alias for the mock, and is unneeded. The code can be refactored to remove this (and maybe change the fixture name to `mock_twilio_client` for less characters).